### DISCUSSION TITLE
Target #84 offset:path('m640,240V0

### DISCUSSION THREAD

**Discussion by Esger** [cssbattle/forum#118] [Q&A] [answered]
Can someone explain how the offset:path in the top open solution works?

- `m640` means move 640 to the right  —  is this with respect to the element upper left or center?
- `240` means move 240 down?
- `V0` means vertical line — at 0 to the right of te starting point?
- how does this vertical offset path make the elements rotate?

**Comment by SelenIT** [comment, marked as answer]
By default, the `offset` property affects the position of the center of the element (default value of the `offset-anchor` sub-property, which is pretty similar to `transform-origin`) and rotates the element so that it is “facing” the direction of the first segment of the offset path (initial direction is “to the right”). The value of the `path()` function uses the syntax of the SVG `d` attribute, so in this case it’s a verical line starting at 640px right, 240px down and going up to 640px right, 0px (uppercase V means vertical line to the specified absolute y-coordinate). So this offset moves the center of the element to (640,240) and rotates it “face up” (i.e. by 90deg CCW).

**Reply by Esger** [reply]
Thanks for your explanation :)
